File: java/innerClass/inheritence/p2.java
class Parent{
	Parent(int a){
		System.out.println("parent");
	}
}

class Child extends Parent{
	Child(int a){

		System.out.println("child");

	}
}

class Test{
	public static void main(String args[]){

		Child c = new Child(10);



	}
}

File: CPP/class/p2.cpp
#include<iostream>
using namespace std;

class Parent{
	public:
		string address;
	

};

class Child:public Parent{

	protected:
	string fun(){

		cout<<address<<endl;
		return address;
	}	

};


int main(){
	Child c;


	
	cout<<c.fun();


}
